src/sentry/lang/java/processing.py

```diff
@@ -248,4 +314,8 @@ def process_jvm_stacktraces(symbolicator: Symbolicator, data: Any) -> Any:
         raw_exc["module"] = exc["module"]
         raw_exc["type"] = exc["type"]
 
+    classes = response.get("classes")
+    new_attachments = _deobfuscate_view_hierarchies(attachments, classes)
+    attachment_cache.set(cache_key, attachments=new_attachments, timeout=CACHE_TIMEOUT)
```

Comment:
instead of re-setting *all* the attachments, you could only overwrite the view hierarchy ones.

Reply:
How? I was under the impression I had to set the entire `attachments` list at once here.